src/ipfsd-daemon.js

```diff
@@ -111,41 +113,50 @@ class Daemon {
   /**
    * Initialize a repo.
    *
-   * @param {Object} [initOpts={}]
-   * @param {number} [initOpts.keysize=2048] - The bit size of the identiy key.
-   * @param {string} [initOpts.directory=IPFS_PATH] - The location of the repo.
-   * @param {string} [initOpts.pass] - The passphrase of the keychain.
+   * @param {Object} [initOptions={}]
+   * @param {number} [initOptions.bits=2048] - The bit size of the identiy key.
+   * @param {string} [initOptions.directory=IPFS_PATH] - The location of the repo.
+   * @param {string} [initOptions.pass] - The passphrase of the keychain.
    * @param {function (Error, Node)} callback
    * @returns {undefined}
    */
-  init (initOpts, callback) {
+  init (initOptions, callback) {
     if (!callback) {
-      callback = initOpts
-      initOpts = {}
+      callback = initOptions
+      initOptions = {}
```

Comment:
Does this mean that is mandatory to always pass a initOptions object? There should be a check to see if initOptions is a func.

Reply:
we're checking for `!callback` which is not consistent with how do it elsewhere (leftover from prev codebase), I'll change it. Good catch.